Sources_v3/StreamChatUI/ChatChannelList/ChatChannelAvatarView.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatChannelAvatarView<ExtraData: UIExtraDataTypes>: AvatarView {
+    // MARK: - Properties
+    
+    public var channel: _ChatChannel<ExtraData>? {
+        didSet { updateContent() }
+    }
+    
+    // MARK: - Overrides
+    
+    override open func setupLayout() {
+        widthAnchor.constraint(equalTo: heightAnchor, multiplier: 1).isActive = true
+        embed(imageView, insets: .init(
+            top: 0,
+            leading: 0,
+            bottom: 0,
+            trailing: directionalLayoutMargins.trailing
+        ))
```

Comment:
This is confusing. The avatar should either pin to margins or to edges. The stack view that includes this view should put proper spacing around it.